README.md

```diff
@@ -352,6 +352,25 @@ The default port the Pushgateway is listening to is 9091. The path looks like:
 
         curl -X PUT http://pushgateway.example.org:9091/api/v1/admin/wipe
         
+## Management API
+
+Pushgateway provides a set of management API to ease automation and integrations.
+
+* Available endpoints:
+ 
+| HTTP_METHOD |  PATH | DESCRIPTION |
+| :-------: | :-----| :----- |
+| GET    | /-/healthy |  Return 200 whenever the Pushgateway is healthy. |
+| GET    | /-/ready |  Return 200 whenever the Pushgateway is ready. |
+
+* The following endpoints are disabled by default and can be enabled via the `--web.enable-lifecycle` flag.
+
+| HTTP_METHOD |  PATH | DESCRIPTION |
+| :-------: | :-----| :----- |
+| PUT    | /-/quit |  Triggers a graceful shutdown of Pushgateway. |
+
+An alternative way trigger a graceful shutdown is by sending a `SIGTERM` to the Pushgateway process.
```

Comment:
Grammar is a bit off here. How about a simpler formulation like: "Alternatively, a graceful shutdown can be triggered by sending a `SIGTERM` to the Pushgateway process."

Reply:
Actually you're right, I copied it from the Prometheus documentation I think I will fix the text there as well.

https://prometheus.io/docs/prometheus/latest/management_api/#quit